packages/cozy-harvest-lib/package.json

```diff
@@ -31,7 +31,7 @@
     "cozy-bi-auth": "0.0.19",
     "cozy-doctypes": "^1.80.0",
     "cozy-logger": "^1.7.0",
-    "cozy-ui": "^45.3.0",
+    "cozy-ui": "47.4.0",
```

Comment:
Please use a ^ here otherwise the version is unnecessarily restrictive.

- In apps we pin libraries
- In library we use the caret to be more lenient and allow an app to use (hopefull) only 1 version that satisfies all the dependency ranges of libraries.

Reply:
Thks, It's changed